qt/python/mantidqt/project/projectsaver.py

```diff
@@ -0,0 +1,75 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+from json import dump
+from os import makedirs
+from os.path import isdir
+
+from mantidqt.project import workspacesaver
+from mantid import logger
+
+
+ENCODED_FILE_NAME = "mantidsave.project"
```

Comment:
I know I have said we should change how this works above but I'd thought would also comment that in is generally considered good practice to follow DRY (do not repeat yourself) principles so we would not want to define this twice as we have currently.

Reply:
Agreed it was temporary till further discussed anyways :+1: